lib/model/query/auth.js

```diff
@@ -78,7 +78,6 @@ const verbsOn = (actorId, actee) => ({ allFirst }) => {
   `);
 };
 
-
```

Comment:
Hardly worth a mention, but a newline was removed here.

Reply:
Thanks!  Always worth a mention if git history is valuable.